query_object.rb

```diff
@@ -1,45 +1,58 @@
-module YourSuperModule
+module ParamsMethods
```

Comment:
Module name should have `business meaning` but not technical details

Reply:
Fixed. Now is it better?